wadm/lib/supervisor.ex

```diff
@@ -8,6 +8,12 @@ defmodule Wadm.Supervisor do
   def init(_opts) do
     config = Vapor.load!(Wadm.ConfigPlan)
 
+    topologies = [
+      wadmcluster: [
+        strategy: Cluster.Strategy.Gossip
```

Comment:
Is this using the multicast you told me about before? If so, will we do a follow up to use something NATS based?

Reply:
Yeah. There's a stack of already-available cluster strategies (including multiple DNS-based and a k8s-based). We can write our own NATS-based one if we see the need to expand on what comes "stock"